### ISSUE TITLE
'watchr.watch' is not a function

### ISSUE THREAD

**Issue by MissFerrety** [justinsisley/mjml-utils#6]
The build command works great, but the watch throws an error claiming that `.watch` is not a function that you can use with Watchr. Reviewing the documentation, `.open` sounds like the correct usage. I am not adept at this type of coding, so I couldn't get Watchr to work after messing around with it for some time.

However, I did manage to get a hotfix working with `fs.watch`:

```const fs = require('fs');```

```
module.exports = (inputDir, outputDir, extension) => {
  const sourceDir = path.join(process.cwd(), inputDir);

  fs.watch(sourceDir, (eventType, filename) => {
    if (filename) {
      if( 'change' == eventType ){
        onChange(sourceDir+'/'+filename, outputDir, extension);
      }
    } else {
      console.log('An error occurred.');
    }
  });
};

```

**Comment by justinsisley** [comment]
Thanks for finding this. It sounds like many users (including me) aren't using the "watch" functionality or this should have been found earlier.

I've been using [chokidar](https://github.com/paulmillr/chokidar) for these types of tasks lately. I'll swap out watchr for chokidar and get this working out of the box again.

**Comment by justinsisley** [comment]
This should be resolved in v2.0.3. Let me know if you continue to have problems with the "watch" task.

**Comment by MissFerrety** [comment]
Finally back to my workstation and installed the update. Works great. Thanks!
